[SRVKE-1273] Adding little notes on the Logging.level (#59705)
openshift/openshift-docs: modules/serverless-install-kafka-odc.adoc

<6> Defines the number of partitions of the Kafka topics, backed by the `Broker` objects. The default is `10`.
<7> Defines the replication factor of the Kafka topics, backed by the `Broker` objects. The default is `3`.
<8> Enables developers to use a Kafka sink in the cluster.
<9> Defines the log level of the Kafka data plane. Allowed values are `TRACE`, `DEBUG`, `INFO`, `WARN` and `ERROR`. The default value is `INFO`.
+
[NOTE]
====
The `replicationFactor` value must be less than or equal to the number of nodes of your Red Hat AMQ Streams cluster.

> For `logging.level` value it is not recommended to use `DEBUG` or `TRACE` in production since it's too verbose and degrades performance.

Reviewer: ```suggestion
For  `logging.level` it is not recommended to use `DEBUG` or `TRACE` in production because the output is verbose and can degrade performance.
```
